vision/text/detect_words.py

```diff
@@ -21,7 +21,8 @@ class TextDetector:
     """
 
     def __init__(self):
-        self.text = "модулииртибот"
+        self.text = np.array(["модули", "иртибот"])
+        self.tessdata = dict()
```

Comment:
Best practice would be to initialize a dictionary like this:
`self.tessdata: dict = {}`
to emphasize that it is empty while also displaying the type.